test/controllers/home_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class HomeControllerTest < ActionController::TestCase
+  test "should get home" do
```

Comment:
controller tests can be deceptive. If you had an error in your homepage view that would cause your app to 500, would this test catch it?